app/controllers/home/oauths_controller.rb

```diff
@@ -10,17 +8,17 @@ def oauth
   def callback
     provider = auth_params[:provider]
     if @user = login_from(provider)
-      redirect_to root_path, notice: (t 'log_in_is_successful_provider_notice',
+      redirect_to trainer_path, notice: (t 'log_in_is_successful_provider_notice',
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.